python/ray/serve/_private/autoscaling_policy.py

```diff
@@ -23,17 +26,19 @@ def _create_policy(self):
         if self.config:
             self.policy = self.config.get_policy()
 
-    def should_autoscale(self) -> bool:
+    def is_autoscaling_policy_enabled(self) -> bool:
```

Comment:
much better :)

Reply:
I think we can drop "autoscaling" from it, provided its `AutoscalingManager` to just `is_policy_enabled`